### PR TITLE
Argument support nixbang

### PR DESCRIPTION
**Pull request by polynomial**
Adding support for arguments passed to nixbang scripts. This would make my life awesome, and I hope not make everyone else's life totally horrible.

### CODE REVIEW

File: nixbang [hunk 1]
```diff
@@ -37,8 +37,13 @@ def main():
...
         return
     options = read_file(target)
-    command = '%s %s' % (options['command'], target)
+    args = ''
+    if len(sys.argv) > 1:
+       args = ' '.join(sys.argv[2:])
+
+    command = '%s %s %s' % (options['command'], target, args)
     subprocess.call(['nix-shell', '--command', command,
+                     '--args', args,
```
Comment (madjar) [inline nixbang]: Aren't arguments passed twice ?

**Comment by madjar** [comment]
I'm so sorry, I though this was an issue instead of a pull request, and I let it sleep here for weeks.

I've added a comment to the code, since there's something I'm not sure I understand.

File: nixbang (hunk 1 above)
Comment (polynomial) [inline nixbang]: Oh, yes woops. I was doing it with that first and changed it. Fixed in an updated commit! Thank you.

**Comment by polynomial** [comment]
@madjar no worries, thanks for looking (: I updated per your comment.

**Comment by madjar** [comment]
Thanks! I'll update it in nixpkgs today if I can.

**Comment by madjar** [comment]
Here you go: https://github.com/NixOS/nixpkgs/commit/7372c5434f9adb22cabf6c6c1e66440db3f283d6

**Pull request merged**